src/sonic-db-cli/tests/Makefile.am

```diff
@@ -0,0 +1,15 @@
+#INCLUDES = -I $(top_srcdir)
+
+bin_PROGRAMS = tests

```

Comment:
>tests [](http://example.com/codeflow?start=15&length=5)

Discussed offline. Let's move testcase into sonic-swss-common repo.

You can use real redis instances in unit test.
You can use either googletest or pytest frameworks.
You can use `sonic-db-cli` as a process.

I am thinking about reuse the same set of testcases in order to test the behavior of old python version of sonic-db-cli. The co-test could last for a while until the python version is completely deprecated and removed from all applications.

Reply:
WIll also move sests to sonic-swss-common and submit in another PR.